AuxTel/Troubleshooting/ATCS/ATHexapod-fails-to-enable-with-the-rest-of-ATCS.rst

```diff
@@ -0,0 +1,93 @@
+  line 518, in do_start    await self._do_change_state(data, "start", [State.STANDBY], State.DISABLED)  File "/opt/lsst/software/stack/miniconda/lib/python3.10/site-packages/lsst/ts/salobj/base_csc.py", 
+  line 830, in _do_change_state    await getattr(self, f"end_{cmd_name}")(data)  File "/opt/lsst/software/stack/miniconda/lib/python3.10/site-packages/lsst/ts/ATHexapod/csc.py", 
+  line 206, in end_start    raise e  File "/opt/lsst/software/stack/miniconda/lib/python3.10/site-packages/lsst/ts/ATHexapod/csc.py", 
+  line 202, in end_start    await self.controller.connect() File "/opt/lsst/software/stack/miniconda/lib/python3.10/site-packages/lsst/ts/ATHexapod/controller.py", 
+  line 97, in connect    self.reader, self.writer = await asyncio.wait_for(  File "/opt/lsst/software/stack/miniconda/lib/python3.10/asyncio/tasks.py", 
+  line 445, in wait_for    return fut.result()  File "/opt/lsst/software/stack/miniconda/lib/python3.10/asyncio/streams.py", 
+  line 48, in open_connection    transport, _ = await loop.create_connection(  File "/opt/lsst/software/stack/miniconda/lib/python3.10/asyncio/base_events.py", 
+  line 1076, in create_connection    raise exceptions[0]  File "/opt/lsst/software/stack/miniconda/lib/python3.10/asyncio/base_events.py", 
+  line 1060, in create_connection    sock = await self._connect_sock(  File "/opt/lsst/software/stack/miniconda/lib/python3.10/asyncio/base_events.py", 
+  line 969, in _connect_sock    await self.sock_connect(sock, address)  File "/opt/lsst/software/stack/miniconda/lib/python3.10/asyncio/selector_events.py", 
+  line 501, in sock_connect    return await fut  File "/opt/lsst/software/stack/miniconda/lib/python3.10/asyncio/selector_events.py", 
+  line 541, in _sock_connect_cb    raise OSError(err, f'Connect call failed {address}')OSError: [Errno 113] Connect call failed ('139.229.170.48', 50000)
+
+
+Also, trying to ping the controller at `athexapod.cp.lsst.org <athexapod.cp.lsst.org>`__ results in no response.
+
+
+
+.. _ATHexapod-fails-to-enable-with-the-rest-of-ATCS-Solution:
+
+Solution
+========
+
+The connection to the ATHexapod can be recovered by manually rebooting the ATHexapod controller box located in the ATCS cabinet in the pier of the AuxTel dome.
+
+
+.. _ATHexapod-fails-to-enable-with-the-rest-of-ATCS-Procedure-Critical-Step-1:
+
+1. Announce in the slack channel #summit-anounce that you are going to the AuxTel building to reboot the ATHexapod Controller.
+   
+2. On the first level, or pier of the AuxTel dome, open the Main ATCS cabinet and press the off button at the C-887 Hexapod Controller, that's located at the bottom of the cabinet.
+
+   .. figure:: ./_static/ATHexapod_Controller.jpg
+       :name: ATHexapodCtrl
+       :scale: 20 %
+
+       ATHexapod Controller.
+
+3. Wait 3 minutes.
+4. Turn the controller on again.
```

Comment:
This is a physical button. Consider including the follwoing formation -- 
1. location of the ON/OFF button 
2. I also feel it should be ``On`` since its a variable state (@isotuela what do you think ? )
3.  on --- > ``On`` (Position "--" of the black horizontal flip button)